src/main/java/edu/cornell/kfs/concur/service/ConcurRequestExtractFileService.java

```diff
@@ -0,0 +1,15 @@
+package edu.cornell.kfs.concur.service;
+
+import org.kuali.kfs.sys.batch.InitiateDirectory;
+
+public interface ConcurRequestExtractFileService extends InitiateDirectory {
+
+    public boolean requestExtractHeaderRowValidatesToFileContents(String fileName);
```

Comment:
public modifier can be removed from these methods as they are public by default.